.github/workflows/reusable-release.yml

```diff
@@ -153,7 +153,14 @@ jobs:
           distribution: corretto
           java-version: 17
       - name: Generate artifacts
-        run: sbt ";set ThisBuild / publishTo := Some(Resolver.file(\"foobar\", file(\"$LOCAL_ARTIFACTS_STAGING_PATH\"))); +publish"
+        run: |
+          cat << EndOfFile > sbt-commands.txt
+          set every sonatypeProjectHosting := Some(xerial.sbt.Sonatype.GitHubHosting("$GITHUB_REPOSITORY_OWNER", "${GITHUB_REPOSITORY#*/}", "${{ needs.identifiers-for-signing-key.outputs.key_email }}"))
```

Comment:
Note that for setting `sonatypeProjectHosting`, neither using the `ThisBuild` or `Global` would work:

```
set ThisBuild / sonatypeProjectHosting
```

or

```
set Global / sonatypeProjectHosting
```

...both gave an error like this:

```
[info] Defining ThisBuild / sonatypeProjectHosting
[info] The new value will be used by no settings or tasks.
```
https://github.com/guardian/play-googleauth/actions/runs/7140777885/job/19446776922#step:4:29

...using `set every` (as in `set every sonatypeProjectHosting`) is something I never had
 even heard of before, but it works, and sets the subproject settings.